docs/quickstart.md

```diff
@@ -0,0 +1,189 @@
+
+Below is an example work flow where a business analyst accesses this list of available services by calling 
+
+`GET https://example.com/cds-services` 
+
+and then configures them in the system. 
+
+![business analyst configuration](images/analyst-configuration-spec.png)
+
+This image captures a business analyst reviewing services from one CDS provider. A business analyst may review services from multiple providers and configure appropriate services per user profiles.
+
+### Invoke service on patient-view hook
+The patient-view hook is invoked when a patient chart is opened. It's one of the most basic since the logic doesn't have any prior workflow dependencies. The service called on the patient-view hook could be dependent on patient characteristics, for example: sex, problems in problems list, active medications, etc. The current version of the CDS Hooks specification allows the EHR to decide which characteristics to consider. 
+
+### Support for FHIR resources on request or prefetch
+Often a CDS service will require additional information from the EHR to perform the decision support logic, or determine the appropriate SMART app to return. Prefetch provides the EHR the capability to pass a resource when invoking a service. For example, with a patient resource included a service could do a geography search for potential environmental risk factors. Below is an example request invoked on patient-view with a patient included: (***fix server, not working for me)
+
+```json
+{
+   "hookInstance" : "23f1a303-991f-4118-86c5-11d99a39222e",
+   "fhirServer" : "http://hooks.smarthealthit.org:9080",
+   "hook" : "patient-view",
+   "redirect" : "http://hooks2.smarthealthit.org/service-done.html",
+   "user" : "Practitioner/example",
+   "context" : [],
+   "patient" : "1288992",
+   "prefetch" : {
+      "patientToGreet" : {
+         "response" : {
+            "status" : "200 OK"
+         },
+         "resource" : {
+            "resourceType" : "Patient",
+            "gender" : "male",
+            "birthDate" : "1925-12-23",
+            "id" : "1288992",
+            "active" : true
+         }
+      }
+   }
+}
+```
+
+
+In some cases, additional information beyond what is included in the prefetch maybe required. The CDS service can request additional information using the FHIR REST APIs:
+
+`GET [base]/AllergyIntolerance?patient=[id]`
+
+It is recommended FHIR servers implement, CDS Services follow, the guidance in the [Argonaut Data Query Guide (DSTU2)](http://www.fhir.org/guides/argonaut/r2/) or [HL7 US Core (STU3)](http://hl7.org/fhir/us/core/index.html) implementation guides when retrieving additional resources. Each profile page within these implementation guides includes queries FHIR servers are required to support. 
```

Comment:
Do we need to call out the Argonaut or US Core profile? While this is certainly applicable to the US market, it may not be in the rest of the world.

Reply:
I updated reference to make it clear these profiles are US examples. I think there is value in encouraging all implementers to look to profiles before rolling their won.